src/aesthetics.jl

```diff
@@ -169,7 +171,7 @@ function assert_aesthetics_undefined(who::AbstractString, aes::Aesthetics, vars:
     defined_vars = intersect(Set(vars), defined_aesthetics(aes))
     if !isempty(defined_vars)
         error(@sprintf("The following aesthetics are defined but incompatible with %s: %s\n",
-                       who, join(undefined_vars, ", ")))
+                       who, join(defined_vars, ", ")))
```

Comment:
because `undefined_vars` is not defined in that function.  calling it throws an error.  it's clearly a typo.  compare to the function immediately preceding this one.

also, let's hold off on merging for a couple days just to make sure all is well.  i still, for example, want to understand why `src/precompile` is necessary.  Gadfly and Compose are the only packages i've seen that do this.